src/commands/workflow/run.ts

```diff
@@ -0,0 +1,47 @@
+import { Flags } from "@oclif/core";
+
+import * as ApiV1 from "@/lib/api-v1";
+import BaseCommand from "@/lib/base-command";
+import { ApiError } from "@/lib/helpers/error";
+import { commaSeparatedArray, jsonStr } from "@/lib/helpers/flag";
+import {
+  formatErrorRespMessage,
+  isSuccessResp,
+  withSpinner,
+} from "@/lib/helpers/request";
+
+export default class WorkflowRun extends BaseCommand {
+  static flags = {
+    environment: Flags.string({ default: "development" }),
+    recipients: commaSeparatedArray({
+      required: true,
+      aliases: ["recipient"],
+      multiple: true,
```

Comment:
Since we are accepting a comma separated string for an array, I'm not sure if we need to set `multiple: true` here.